.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,3 +7,4 @@
 - [ ] The page has 8 or fewer examples.
 - [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
 - [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
+- [ ] The page includes a web link (if applicable)
```

Comment:
```suggestion
- [ ] The page includes a web link in the description (if applicable).
```

Reply:
Good idea. Maybe "in the main description", just to be extra clear?

Reply:
Right, makes sense.